.jenkins/pytorch/common_utils.sh

```diff
@@ -145,6 +145,10 @@ function install_functorch() {
   popd
 }
 
+function test_functorch() {
```

Comment:
why move the test_functorch function to common_utils from test.sh?

Reply:
I want to use this in https://github.com/pytorch/pytorch/blob/master/.jenkins/pytorch/macos-test.sh when I add macos tests and this file imports common_utils.sh